roles-in-tech/security.md

```diff
@@ -68,33 +89,26 @@ These are some misconceptions that people have in their mind but it is not alway
 
 - **Small organizations will not be attacked** - Major organizations assume that they are small and will not be targeted as we are small company. The research shows that 70% of cyber attacks target small businesses. So today for everyone it is very important to take care of its security if its data is worth value for money, whether it is a website for fun or it is a huge organization.
 
-
-OWASP releases a regular list of the [top 10 most critical web application security risks](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_Project). Here are the 2017 highlights:
-- Injection: validate everything before you give it to an interpreter! Here we focus on JavaScript sanitization.
-- Broken Authentication and Session Management: practice this by using the above libraries. Better than passwords, try SAML.
-- XSS and CRSF: XSS is basically injection in the DOM, and you can't prevent CRSF until you've tackled all the XSS bugs. Try it at home with [Google](https://xss-game.appspot.com/) and [Excess XSS](http://excess-xss.com/).
-- Encryption: sensitive data should be encrypted in transit and at rest. Also, you are not a mathematician; never try to roll your own encryption.
-
 ### Guided Practice
-
-- Apprentice will use popular libraries to add authentication and session management to the Pinterest clone project. Perhaps using Google or [OpenID](http://docs.identityserver.io/en/release/quickstarts/7_javascript_client.html#refjavascriptquickstart) for SAML auth is better than trying Passport.
-  - Also consider [Auth0](/oauth/o-auth.md).
-- Next, you can validate user input with Parsley and validate fields with Validator.
-- Use DOMPurify to prevent XSS.
+- Talk to as many professionals as you can [National Cyber Security Centre](https://live.ncsc.gov.uk/topics/new-talent) is worth for who lives in UK
```

Comment:
This line is pretty strange... please look at it in the markdown preview and change it.